pkg/docker/docker.go

```diff
@@ -1053,10 +1053,12 @@ func (d *stiDocker) RunContainer(opts RunContainerOptions) error {
 		// Return an error if the exit code of the container is
 		// non-zero.
 		glog.V(4).Infof("Waiting for container %q to stop ...", container.ID)
-		exitCode, err := d.client.ContainerWait(context.Background(), container.ID)
-		if err != nil {
-			return fmt.Errorf("waiting for container %q to stop: %v", container.ID, err)
-		}
+		// FIXME
+		exitCode := 0
+		// exitCode, err := d.client.ContainerWait(context.Background(), container.ID)
```

Comment:
This is the only bit that is left for me to finish it. It looks like they've changed it significantly and worst thing is that the current conditions (`WaitConditionNotRunning`, `WaitConditionNextExit`, `WaitConditionRemoved`) don't fit us. I was hoping the `WiatConditionNextExit` will, but if the container exits in between this will block until next exit, which is not quite right. I'll dig a bit more into it, and will see what's possible and what not.

Reply:
@bparees ^